src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -1312,8 +1317,18 @@ def load_arguments(self, _):
             action="store_true",
             is_preview=True,
         )
-        c.argument("enable_cost_analysis", action="store_true")
-        c.argument("disable_cost_analysis", action="store_true")
+        c.argument(
+            "enable_fqdn_policy",
+            action="store_true",
+            is_preview=True,
+        )
+        c.argument(
+            "disable_fqdn_policy",
+            action="store_true",
+            is_preview=True,
+        )
+        c.argument("enable_cost_analysis", is_preview=True, action="store_true")
+        c.argument("disable_cost_analysis", is_preview=True, action="store_true")
```

Comment:
seems there is a problem on how you handled merge conflict, `is_preview=True` is removed in PR #7851

Reply:
Thanks for the catch